nginx.conf.example

```diff
@@ -0,0 +1,85 @@
+# nginx -p ./ -c nginx.conf
+
+# display_name=Internet WWW11, access_url=http://localhost:8080/internet/www11, path=internet/www11, year=2015
+# display_name=Internet WWW12, access_url=http://localhost:8080/internet/www12, path=internet/www12, year=2016 - nie ma plików bo zaginęły :P
+# display_name=Internet WWW13, access_url=http://localhost:8080/internet/www13, path=internet/www13, year=2017
+# display_name=Internet WWW14, access_url=http://localhost:8080/internet/www14, path=internet/www14, year=2018
+
+worker_processes auto;
+daemon off;
+pid nginx.pid;
+error_log /dev/stdout;
+
+events {
+	worker_connections 768;
+	# multi_accept on;
```

Comment:
?

Reply:
@krzys-h file. This example is not really important. Probably this is a default (so uncommenting wouldn't change anything.

Reply:
This is just a heavily modified default config from Ubuntu, I left everything that wasn't important as-is